src/components/Nav/Nav.js

```diff
@@ -1,8 +1,80 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Nav.scss";
 
-export class Nav extends Component {
+class Nav extends Component {
   render() {
-    return <div></div>;
+    return (
+      <nav className="Nav">
+        <div className="container">
+          <div className="logoWrap">
+            <img src="./images/logo.png" alt="logo" className="logo" />
+          </div>
+          <div className="navContainer">
+            <div className="topLineWrap">
+              <form className="searchBarWrap">
+                <input className="searchBar" type="text" placeholder="Search" />
+                <button className="searchBarBtn">
```

Comment:
```suggestion
                <button type="button" className="searchBarBtn">
```
button 태그에 type button을 부여하는 이유를 찾아서 댓글로 남겨주세요!

Reply:
찾아보고 블로그도 썼습니다!!! 
button은 기본값으로 type="submit"을 가지고 있습니다. form 태그 내부의 button 역시 기본값은 submit인데 이 경우 버튼을 눌렀을 때 의도치 않은 동작 즉, 버튼은 submit을 수행하게 됩니다. 
하지만 type="button"일 경우 버튼은 특정 이벤트를 걸어주지 않는한 아무동작도 하지 않습니다. 그렇기에 버튼의 의도치 않은 동작을 예방할 수 있어서 꼭 type="button"을 써야합니다. 

또한 form 태그 밖의 button은 기본값으로 submit을 가지고 있지는 않지만 sementic tag로서 form 태그가 권장되기 때문에 반드시 type="button"을 쓰는 습관을 들여야합니다. 

제가 기본기가 부족했음을 느끼고 있습니다. 리뷰 감사합니다😍